client/pom.xml

```diff
@@ -50,5 +50,58 @@
 			<artifactId>jzlib</artifactId>
 			<version>1.1.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-server</artifactId>
+			<version>9.2.14.v20151106</version>
```

Comment:
jetty-server is already in the test classpath (see parent pom).

We depend on Jetty 9.3 (latest), not 9.2. I'm not inclined to downgrade, as upgrading from 9.2 to 9.3 let me uncover some bugs.